internal/storage/bucket_handle_test.go

```diff
@@ -407,13 +408,59 @@ func (t *BucketHandleTest) TestListObjectMethodForMaxResult() {
 	AssertEq(TestObjectName, fourObj.Objects[3].Name)
 	AssertEq(nil, fourObj.CollapsedRuns)
 
+	// Validate that 2 objects are listed when MaxResults is passed 2.
 	AssertEq(nil, err2)
 	AssertEq(2, len(twoObj.Objects))
 	AssertEq(TestObjectRootFolderName, twoObj.Objects[0].Name)
 	AssertEq(TestObjectSubRootFolderName, twoObj.Objects[1].Name)
 	AssertEq(nil, twoObj.CollapsedRuns)
 }
 
+func (t *BucketHandleTest) TestListObjectMethodWithMissingMaxResult() {
+	// We have 4 objects in the fakestorage.
+	fourObjWithoutMaxResults, err := t.bucketHandle.ListObjects(context.Background(),
+		&gcs.ListObjectsRequest{
+			Prefix:                   "",
+			Delimiter:                "",
+			IncludeTrailingDelimiter: true,
+			ContinuationToken:        "",
+			//MaxResults:               2, // Missing MaxResults
```

Comment:
we dont add lines and then comment it out. the testcase name itselft tell max result is missing.

Reply:
removed.